preview-src/index.adoc

```diff
@@ -192,7 +191,7 @@ This section does not have any edition or status labels at the start.
 
 * This is the first item in a list.
 
-* [.edition]#{community}# This is the second item in a list.
+* [.community]#{community}# This is the second item in a list.
```

Comment:
The label "edition" can have one of two values: Enterprise or Community 
It looks like we are defining a new label for community instead. Is that correct?

Reply:
Hi @amarantha-k ,

I'm defining a community class for Community label div. In a new style guide community-level button UI showing in gray style, So I put a community class to differentiate it from other page statuses/edition button.